documentation/running_bolt_commands.md

```diff
@@ -27,6 +27,26 @@ Invoke-BoltCommand -Command 'Get-Location' -Targets servers
 > 🔩 **Tip:** If a command contains spaces or special shell characters, wrap
 > the command in single quotation marks.
 
+### Run a quoted command
+
+If you need to run a command with quotes inside, there are a few considerations to keep in mind. You will need properly escape the quotes according to the whether you are running in `bash` or in `powershell`.
```

Comment:
```suggestion
If you need to run a command that uses quotation marks, you must properly escape the quotations. The way you escape the quotations depends on whether you're using Bash or PowerShell.
```

Reply:
Does the Bash stuff apply to all *nix style shells? Just wondering if we should sub Bash for *nix